### DISCUSSION TITLE
JavaScript/ HTML

### DISCUSSION THREAD

**Discussion by tobiashyu** [thebaselab/codeapp#479] [Q&A]
Is there a way to view the website side by side with our code? As well as the console? Similar to what Spck Editor can do? 
![3A77E2CB-FB28-46E2-B162-96587650F0D3](https://user-images.githubusercontent.com/75548448/172099790-e80f0351-736b-4bc4-9622-bb1d9f49edf5.jpeg)

**Comment by luni-moon** [comment]
I see this would be nice for iPadOS users! +1

**Reply by tobiashyu** [reply]
It would be a nice addition, right now I have to use another app to get to this, but it’s not the best solution.

**Comment by bummoblizard** [comment]
While there is no browser console yet, iPad has built in support for Split View. You can open Safari side by side by dragging the url on the top of the browser sheet.

**Reply by hg0428** [reply]
You can use Eruda

**Reply by tobiashyu** [reply]
Hi! Not familiar with Eruda, what is it?

**Reply by hg0428** [reply]
> Hi! Not familiar with Eruda, what is it?

Its a console for mobile browsers.
It can be imported via a <script></script> tag, or downloaded via a safari extension: ![image](https://user-images.githubusercontent.com/45984899/173904386-ed56479c-bf64-4e24-a123-d21e684147a0.jpeg)

I use the "Web Inspector" safari extension, it runs on Eruda and gives you access to a console, a web inspector and more.

"Web Inspector" does not unlock the full capability of Eruda though, Eruda has more to offer than what "Web Inspector" takes advantage of.

**Reply by tobiashyu** [reply]
Thank you! I just got this to work! Hopefully there will be a built-in console in the future!
